src/main/java/com/redhat/management/approval/Request.java

```diff
@@ -155,7 +151,7 @@ public String getPostActionHeaders() {
         rhid.getUser().setEmail(SYSADMIN);
         rhid.getUser().setFirst_name(SYSADMIN);
         rhid.getUser().setLast_name(SYSADMIN);
-        return "x-rh-identity=" + createEncodedIdentity(rhid);
+        return String.format("x-rh-identity=%s", createEncodedIdentity(rhid));
```

Comment:
I think simple string concatenation is ok with `+`. `String.format` is good for mixing of static text and variable.